include/pressiodemoapps/impl/mesh_ccu.hpp

```diff
@@ -130,6 +130,10 @@ class CellCenteredUniformMesh
   const y_coords_type & viewY() const{ return m_y; }
   const z_coords_type & viewZ() const{ return m_z; }
 
+  int nx() const{ return m_meshDims[0]; };
+  int ny() const{ return m_meshDims[1]; };
+  int nz() const{ return m_meshDims[2]; };
```

Comment:
these will need to go away because they have no meaning for a generic collection of cells